OracleGoldenGate/21c/install-deployment.sh

```diff
@@ -82,6 +82,11 @@ EOF
     else
         $(run_as_ogg) tar xf /tmp/installer/*.tar -C "${OGG_HOME}"
     fi
+
+    for dir in "${OGG_HOME}/scripts/setup" "${OGG_HOME}/scripts/startup"; do
+        mkdir -p "${dir}"
+        chown -R ogg:ogg "$(dirname "${dir}")"
+    done
```

Comment:
or:
```.sh
mkdir -p '${OGG_HOME}/scipts/"{setup,startup}
chown -R ogg:ogg "${OGG_HOME}/scripts"
```

Reply:
replaced for loop with suggested mkdir single execution